spec/linux/kernel.go

```diff
@@ -42,5 +43,72 @@ func (g *KernelGenerator) Generate() (interface{}, error) {
 		results[key] = str
 	}
 
+	hostInfo, err := host.Info()
+	if err != nil {
+		kernelLogger.Errorf("Failed to get platform information: %s", err)
```

Comment:
I added `return results nil` for compatibility,  because results is not returned when `host.Info` returns error.